packages/devtools/src/devtoolsdriver.ts

```diff
@@ -108,6 +94,51 @@ export default class DevToolsDriver {
         log.trace(`Switching to window handle with id ${pageIds[0]}`)
     }
 
+    private addListener(emitter: EventEmitter, eventName: string, handler: any) {
+        const boundHandler = handler.bind(this)
+        emitter.on(eventName, boundHandler)
+        this.activeListeners.push({ emitter, eventName, boundHandler })
+    }
+
+    private offAllListeners() {
```

Comment:
I'd prefer something more descriptive of what the purpose of this method is.
```suggestion
    private cleanupListeners() {
```

Reply:
done